Realm/RLMDecimal128.h

```diff
@@ -67,6 +73,49 @@ NS_ASSUME_NONNULL_BEGIN
 
 /// Gets if this Decimal128 represents a NaN value.
 @property (nonatomic, readonly) BOOL isNaN;
+
+/// The magnitude of this RLMDecimal128.
+@property (nonatomic, readonly) RLMDecimal128 *magnitude NS_REFINED_FOR_SWIFT;
+
+/// Replaces this RLMDecimal128 value with its additive inverse.
+- (void)negate;
```

Comment:
I don't think we want any mutating functions? NSDecimalNumber doesn't have them and they won't work how users expect them to (i.e. `[foo.decimalProp additionAssignmentWith:num]` won't actually mutate the managed object).

Reply:
They are there more-so for the swift implementation. It should be the case that the managed object is mutated

```
- (void)additionAssignmentWith:(RLMDecimal128 *)decimalNumber {
    auto rhs = RLMObjcToDecimal128(decimalNumber);
    _value+=rhs;
}
```

Reply:
Modifying the `realm::Decimal128` ivar doesn't write to the Realm (and can't, as it doesn't have any association to the Realm it was read from).

Reply:
I've removed the assignment operators but it removes the `SignedNumeric` conformance for `Decimal128`